src/helpers/storybookUtils.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * For usage in Storybook stories to set the default value of a boolean prop.
  */
-export const argsBooleanType = (defaultValue = false) => ({
+export const argsBooleanType: any = (defaultValue = false) => ({
   control: { type: "boolean" },
-  table: { defaultValue: { summary: defaultValue } },
+  table: { defaultValue: { summary: `${defaultValue}` } },
```

Comment:
This may be the weirdest one of all.

Reply:
Ah yes, it's because the `summary` must now be a string so I had to cast it. I will actually make an update to update this to be more correct.